src/Angor/Client/Services/MessageService.cs

```diff
@@ -15,6 +16,7 @@ public class MessageService : IMessageService
         private readonly INetworkService _networkService;
         private readonly IRelayService _relayService;
         private readonly NostrConversionHelper _nostrHelper;
+        private readonly HtmlSanitizer _sanitizer;
```

Comment:
I think this should be wrapped in an interface so we can replace it if we want to

Reply:
yeah I agree